docs/definitions/expression-language.md

```diff
@@ -320,11 +320,11 @@ Examples
 @=getUser().firstName === 'adam'
 ```
 
+
 ## Registered variables:
 
 | Variable             | Description  | Scope|
 |:-------------------- |:------------ |:---- |
-| `typeResolver`       | An object of class `Overblog\GraphQLBundle\Resolver\TypeResolver`| global|
```

Comment:
why this change ?

Reply:
@mcg-web It's a mistake. First I removed `typeResolver` from expressions, then I rerolled this change, but forgot to reroll changes done to the documentation. I'll fix this.